.ci_support/migrations/cuda120.yaml

```
@@ -2,12 +2,8 @@ migrator_ts: 1682985063
 __migrator:
   kind:
     version
-  # Vendor CUDA 12 migrator to use GCC 11.
-  # This is needed to workaround a pybind + GCC 12 + nvcc 12 bug
-  # xref: https://github.com/pybind/pybind11/issues/4606
-  use_local: true
   migration_number:
```

Readding the `use_local` logic that was above

```suggestion
  # Vendor CUDA 12 migrator to use GCC 11.
  # This is needed to workaround a pybind + GCC 12 + nvcc 12 bug
  # xref: https://github.com/pybind/pybind11/issues/4606
  use_local: true
  migration_number:
```